<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botaoDeLogin">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(42, 42, 42);
border-radius: 15px;
border-style: outset;
border-width: 2px;
border-color: black;
font: 25 italic 12pt &quot;Ubuntu&quot;;

</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userLogin">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="placeholderText">
     <string>Digite o seu usuário</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="senhaLogin">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset;
border-width: 2px;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Digite a sua senha</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="CadastroLogin">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>380</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 italic 12pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Não é cadastrado?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoDeCadastrarLogin">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>390</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(42, 42, 42);
border-radius: 15px;
border-style: outset;
border-width: 2px;
border-color: black;
font: 25 italic 12pt &quot;Ubuntu&quot;;

</string>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagemUserLogin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>user.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ImagemSenhaLogin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>senha.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="mensagemDeErroLogin">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
font: 25 italic 12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>ImagemSenhaLogin</zorder>
   <zorder>botaoDeLogin</zorder>
   <zorder>userLogin</zorder>
   <zorder>senhaLogin</zorder>
   <zorder>CadastroLogin</zorder>
   <zorder>botaoDeCadastrarLogin</zorder>
   <zorder>imagemUserLogin</zorder>
   <zorder>mensagemDeErroLogin</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
